nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -65,6 +65,12 @@
           new versions will release.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>services.kubernetes.addons.dashboard</literal> was
+          removed due to it being an outdated version.
+        </para>
+      </listitem>
```

Comment:
So the dashboard itself is still maintained, I'm wondering if we consider updating it rather than removing.

Reply:
I don't think it's been updated since it was added, we've also had an issue about it open for over a year with no action.

Even if we update it now I doubt it will be maintained going forward.

Reply:
In the perfect world, someone (else) could maintain the dashboard as a separate module/package. I personally haven't used the dashboard for years in my clusters.

Generally, I'd love it if the kubernetes module could be split in to smaller modules, maintained by different people with different interests in k8s. And I want to hear from people that use the dashboard provided here today? @saschagrunert do you? :)

On another node, the legacy addon-manager needs to die at some point. The Kubernetes community advises against using it. So we'll need to find a different way of "scheduling" addons soon anyway.

Reply:
> the legacy addon-manager needs to die at some point. The Kubernetes community advises against using it.

If we're going to this I'd rather we remove it all once. I'll switch this PR to a draft for now while we decide.

Reply:
Yeah, sounds reasonable to remove them. :+1:

Reply:
Having looked at the code a bit more it seems removing everything at once will be a largish PR that needs to be well tested. Just removing the dashboard is trivial and an easy merge so I'll leave this PR as it and do the addon manager in another.

Reply:
I'll do a follow up for the addon manager in the next few weeks.

Reply:
Starting to wonder if instead of going through the hassle of removing the addon manager we should just deprecate the current kubernetes module and start over on a new module as we've discussed previously.